### DISCUSSION TITLE
Ansible Galaxy Repo displayed as Username

### DISCUSSION THREAD

**Discussion by kishor578** [community/community#70324] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

I've created an organization on github and added user to that organization. When i login to ansible galaxy am able to namespace by my name but not the org name that i have created in github. Any help would be appreciated. Thanks in advance.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @kishor578 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
